### DISCUSSION TITLE
Ability to disable all e-aliases

### DISCUSSION THREAD

**Discussion by mcmuffindk** [EssentialsX/Essentials#4035] [Ideas and Feature Suggestions] [answered]
The e-alias commands are very annoying and I, therefore, suggest that server administrators should be able to turn off these aliases so that they do not show up in the tab completion list

**Comment by pop4959** [comment, marked as answer]
Should be possible if we decide to merge https://github.com/EssentialsX/Essentials/pull/4026. Otherwise, the only way to disable commands is through commands.yml on the server.

**Reply by mcmuffindk** [reply]
With the commands.yml method, the commands will still show up in the tab completion list
